.travis.yml

```diff
@@ -29,17 +24,64 @@ addons:
     paths:
       - /tmp/thunder-travis-ci
 
+stages:
+  - sanity checks
+  - test
+
+jobs:
+  include:
+    - stage: sanity checks
+      name: Verify makefile, check coding standards, check for deprecated code
+      env: INSTALL_METHOD=composer
+      install:
+        # Install Drush
+        - composer global require drush/drush:^8.1
+        # Install drupalorg_drush module and verify, that makefile is accepted by drupal.org, otherwise we do not need to go any further
+        - drush dl drupalorg_drush-7.x && drush verify-makefile
+        - bash -x -e ./scripts/travis/03-download-thunder.sh
+      before_script: true
+      script:
+        - bash -x -e ./scripts/travis/06-test-source-code.sh
+
+    - stage: sanity checks
+      name: Check config
+      env: INSTALL_METHOD=composer
+      script:
+        - bash -x -e ./scripts/travis/06-run-config-tests.sh
+
+    - stage: test
+      name: Run tests with composer build
+      env: INSTALL_METHOD=composer
+
+    - stage: test
+      name: Run tests with drush_make build
+      env: INSTALL_METHOD=drush_make
+      install:
+        # Install Drush
+        - composer global require drush/drush:^8.1
+        - bash -x -e ./scripts/travis/03-download-thunder.sh
+
+    - stage: test
+      name: Run update test
+      env: INSTALL_METHOD=composer TEST_UPDATE=true TEST_DEPLOYMENT=true
```

Comment:
Yes, we have. Should we bring that back?

Reply:
I don't know. I would like to have Update tests for `drush_make` at least on cron runs. That would lead to 3 => 4 jobs on cron. 😞 
Maybe we can think about excluding some jobs on cron.